std_msgs/Header header
float64[] u0
float64[] x1
float64 status
float64 KKT_res
float64 cpu_time
